packages/wallet-core/db/schemas/1-init.js

```diff
@@ -0,0 +1,37 @@
+/* AUTO GENERATED FILE - DON'T CHANGE THE CONTENT */
+
+export default {
+  "schemaVersion": 1,
+  "schema": [
+    {
+      "name": "test",
+      "primaryKey": "id",
+      "properties": {
+        "id": {
+          "type": "int"
+        },
+        "name": {
+          "type": "string"
+        }
+      }
+    },
+    {
+      "name": "wallet",
+      "primaryKey": "id",
+      "properties": {
+        "id": {
+          "type": "int"
+        },
+        "name": {
+          "type": "string"
+        },
+        "publicKey": {
+          "type": "string"
+        },
+        "privateKey": {
```

Comment:
Are we going to save private keys to db?

Reply:
I think so for this first bets. We could remove it after getting the backup wallet feature ready in next bets.  Also, the idea is to enable encryption in the db to improve security